WooCommerce/Classes/Authentication/Store Creation/StoreCreationCoordinator.swift

```diff
@@ -553,24 +553,49 @@ private extension StoreCreationCoordinator {
     enum Localization {
         enum DiscardChangesAlert {
             static let title = NSLocalizedString("Do you want to leave?",
-                                                 comment: "Title of the alert when the user dismisses the store creation flow.")
+                                                 comment: "Title of the alert when the user dismisses the store creation profiler flow.")
             static let message = NSLocalizedString("You will lose all your store information.",
-                                                   comment: "Message of the alert when the user dismisses the store creation flow.")
+                                                   comment: "Message of the alert when the user dismisses the store creation profiler flow.")
             static let confirmActionTitle = NSLocalizedString("Confirm and leave",
-                                                              comment: "Button title Discard Changes in Discard Changes Action Sheet")
+                                                              comment: "Button to confirm the dismissal of  the store creation profiler flow")
             static let cancelActionTitle = NSLocalizedString("Cancel",
-                                                             comment: "Button title Cancel in Discard Changes Action Sheet")
+                                                             comment: "Button to dismiss the alert on the store creation profiler flow")
+        }
+
+        enum DismissalAlert {
+            static let title = NSLocalizedString(
+                "Ready for setup",
+                comment: "Title of the alert when the user dismisses the store creation flow."
+            )
+            static let message = NSLocalizedString(
+                "If you exit now, you'll be guided to 'Add a Store' as your next step. Continue setup?",
```

Comment:
I don't understand this message. Is this message specifically designed for new users without any stores? (i.e. Present after signup)

Reply:
We are dismissing the alert so if the user comes from the store picker, they will be navigated there and the CTA to Add a store should still be visible. I'm not sure of better wording - would you mind sharing a suggestion to propose to Ann?

Reply:
Thanks for the explanation, Huong!

Considering that the user hasn't entered any info, I would suggest removing this dismissal alert.
1. They won't lose any data if they dismiss. 
2. They can simply use the "Add a store" button to reopen this screen if needed. 

What do you think?

Reply:
I agree that this alert seems redundant. I just removed the alert in b1611cf04209dc93a8b569c8da7fe627165613af - keeping the tracking for the dismissal though.